src/graphics/EInkDisplay2.cpp

```diff
@@ -182,6 +182,9 @@ bool EInkDisplay::connect()
         // Init GxEPD2
         adafruitDisplay->init();
         adafruitDisplay->setRotation(3);
+#if defined(CROWPANEL_ESP32S3_5_EPAPER)
+        adafruitDisplay->setRotation(0);
```

Comment:
I assume calling this twice does no harm?

Reply:
The other displays are oriented differently